Database/SQLite/Simple/FromField.hs

```diff
@@ -1,7 +1,9 @@
 
-{-# LANGUAGE CPP, DeriveDataTypeable #-}
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE DeriveDataTypeable #-}
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeSynonymInstances #-}
```

Comment:
This is extra.

Reply:
Not sure what you mean.
My change removed some of the language extensions that this module was not actully using, and then formatted the rest over one line.

Or do you mean I should remove all unused language extensions (TypeSynonymInstances is not used in this module)? If so, I removed all unused extensions from all modules in the next commit.